src/cascade/input_data/db/mortality.py

```diff
@@ -7,14 +7,19 @@
 
 
 def get_excess_mortality_data(execution_context):
+    """
+    The year range is from start of year to end of year, so these
+    measurements have a year duration. To make point data, take
+    the midpoint of the year.
+    """
     model_version_id = execution_context.parameters.model_version_id
 
     query = """
      SELECT
                 model_version_dismod_id as data_id,
                 location_id,
                 year_start as time_lower,
-                year_end as time_upper,
+                year_end + 1 as time_upper,
```

Comment:
Awesome. You did exactly the same thing I did. Character for character. This feels wrong to me because it's stuck directly in the SQL. But doing it outside the SQL seems silly. I don't know. I guess it doesn't matter.